browser/ipfs/ipfs_tab_helper_unittest.cc

```diff
@@ -287,24 +298,36 @@ TEST_F(IpfsTabHelperUnitTest, ResolveXIPFSPathUrl) {
     SetIPFSResolveMethodPref(ipfs::IPFSResolveMethodTypes::IPFS_GATEWAY);
     GURL gateway = ipfs::GetConfiguredBaseGateway(profile()->GetPrefs(),
                                                   chrome::GetChannel());
-    GURL url = helper->ResolveXIPFSPathUrl("/ipfs/bafy");
-    EXPECT_EQ(url, GURL("ipfs://bafy"));
+    GURL url = helper->ResolveXIPFSPathUrl(
+        "/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi");
+    EXPECT_EQ(
+        url,
+        GURL("ipfs://"
+             "bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi"));
   }
 
   {
     SetIPFSResolveMethodPref(ipfs::IPFSResolveMethodTypes::IPFS_LOCAL);
     GURL gateway = ipfs::GetConfiguredBaseGateway(profile()->GetPrefs(),
                                                   chrome::GetChannel());
-    GURL url = helper->ResolveXIPFSPathUrl("/ipfs/bafy");
-    EXPECT_EQ(url, GURL("ipfs://bafy"));
+    GURL url = helper->ResolveXIPFSPathUrl(
+        "/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi");
+    EXPECT_EQ(
+        url,
+        GURL("ipfs://"
+             "bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi"));
   }
 
   {
     SetIPFSResolveMethodPref(ipfs::IPFSResolveMethodTypes::IPFS_ASK);
     GURL gateway = ipfs::GetConfiguredBaseGateway(profile()->GetPrefs(),
                                                   chrome::GetChannel());
-    GURL url = helper->ResolveXIPFSPathUrl("/ipfs/bafy");
-    EXPECT_EQ(url, GURL("ipfs://bafy"));
+    GURL url = helper->ResolveXIPFSPathUrl(
+        "/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi");
+    EXPECT_EQ(
+        url,
+        GURL("ipfs://"
+             "bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi"));
```

Comment:
If this hash is being used throughout the tests, can it be a constant or `#define` so that's obvious, with the various paths constructed around it?